BridgeAppSDK/SBAUser.swift

```diff
@@ -43,8 +43,8 @@ public final class SBAUser: NSObject, SBAUserWrapper {
     
     let lockQueue = DispatchQueue(label: "org.sagebase.UserLockQueue", attributes: [])
 
-    public func logout() {
-        lockQueue.async {
+    public func resetStoredUserData() {
+        lockQueue.sync {
```

Comment:
Okay... so this is another thing that changed with Swift 3.0
Should be a serial FIFO queue now.